cmd/ssh_server/main.go

```diff
@@ -63,9 +63,26 @@ func main() {
 	defer verification_cleaner.Close()
 
 	// initialize mail sender
-	mail_sender, err := mail.NewMailSender(cfg.Email.ResendKeyPath, cfg.Email.FromEmail, cfg.Email.FromName)
-	if err != nil {
-		log.Fatalf("Could not initialize MailSender: %s", err)
+	var mail_sender mail.MailSender
+	switch cfg.Email.EmailService {
+	case "resend":
+		mail_sender, err = mail.NewResendMailSender(cfg.Email.Resend.ResendKeyPath, cfg.Email.FromEmail, cfg.Email.FromName)
+		if err != nil {
+			log.Fatalf("Could not initialize ResendMailSender: %s", err)
+		}
+	case "smtp":
+		mail_sender = mail.NewSTMPMailSender(
```

Comment:
typo: probably should be `NewSMTPMailSeerver`

Reply:
It's not a typo. Actually, it's not the smtp server, and even it's smtp client.
SMTPMailSender connects to the smtp server and sends the emails through them.
I think "MailSender" is a good postfix, and it's also used in "ResendMailSender".

Reply:
right, it is currently called STMP instead of SMTP... that's the typo (in my comment I intended to write sender instead of server)